evm/hyperliquid.js

```diff
@@ -334,22 +305,43 @@ async function main(options) {
 if (require.main === module) {
     const program = new Command();
 
-    program.name('HyperliquidBlockHelper').description('Script to manage Hyperliquid specific actions');
+    program.name('hyperliquid').description('Hyperliquid chain management commands');
+
+    // Update block size command
+    const updateBlockSizeCmd = program
+        .command('update-block-size')
```

Comment:
```suggestion
        .command('update-block-size <block-size>')
```
see the newer `stellar/its.js` for CLI parsing pattern